ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/qml/SearchView.qml

```diff
@@ -52,6 +53,7 @@ ListView {
                     drawer.close();
                     // launch sample...
                     SampleManager.currentSample = sample;
+                    GAnalytics.postEvent("sample_opened", {"sample name": sample.name, "referrer": "search"});
```

Comment:
Ditto does this need a check? `if (GAnalytics.telemetryEnabled)` ?

There are a lot of these, so I won't keep asking.